charts/rancher-gatekeeper/100.0.1+up3.6.0/Chart.yaml

```diff
@@ -2,9 +2,11 @@ annotations:
   catalog.cattle.io/auto-install: rancher-gatekeeper-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: OPA Gatekeeper
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.24.0-0'
```

Comment:
This was released in 2.6.1 https://github.com/rancher/rancher/issues/34589

According to https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-1, this should have only been tested up to k8s 1.21; are we sure 1.22 and 1.23 are supported?